libcontainer/configs/namespaces_unsupported.go

```diff
@@ -1,3 +1,5 @@
+// +build !linux,!freebsd
```

Comment:
I think doing a not is better so we don't have to list every platform. So, we could keep it as is for now.